.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+    build:
+        docker:
+            - image: circleci/ruby:stretch
+        steps:
+            - checkout
+
+            - run:
+                name: install
+                command: |
+                    gem install jekyll bundler
+                    bundle
+
+            - run:
+                name: build and output to _site
+                command: |
+                    bundle exec jekyll build --baseurl /0/home/circleci/project/_site
+
+            - store_artifacts:
+                path: ~/project/_site
+                destination: html
```

Comment:
mh perhaps this was the culprit, in reverting https://github.com/bids-standard/bids-website/pull/87/commits/499e60f351776f630c606a9fcf88cc6615a01dd2 you (accidentally?) re-introduced the `destination` key ... which overrides the default destination of `circleci/projects`